### DISCUSSION TITLE
where is my page stored

### DISCUSSION THREAD

**Discussion by wind4gis** [toeverything/AFFiNE#4616] [Q&A] [answered]
i don t konw where is my files stored, i need a tool to sync those files in cloud, so i can deal with it in another computer..

**Comment by fourdim** [comment, marked as answer]
https://github.com/electron/electron/blob/main/docs/api/app.md#appgetpathname

### `app.getPath(name)`

* `name` string - You can request the following paths by the name:
  * `home` User's home directory.
  * `appData` Per-user application data directory, which by default points to:
    * `%APPDATA%` on Windows
    * `$XDG_CONFIG_HOME` or `~/.config` on Linux
    * `~/Library/Application Support` on macOS
  * `userData` The directory for storing your app's configuration files, which
    by default is the `appData` directory appended with your app's name. By
    convention files storing user data should be written to this directory, and
    it is not recommended to write large files here because some environments
    may backup this directory to cloud storage.
  * `sessionData` The directory for storing data generated by `Session`, such
    as localStorage, cookies, disk cache, downloaded dictionaries, network
    state, devtools files. By default this points to `userData`. Chromium may
    write very large disk cache here, so if your app does not rely on browser
    storage like localStorage or cookies to save user data, it is recommended
    to set this directory to other locations to avoid polluting the `userData`
    directory.

You might want to backup the whole `userData` folder. But still it does not ensure the backup will work in the future version. It is recommended to install the same version when you backup, upgrade to the latest version, and backup again.
